<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mail</class>
 <widget class="QDialog" name="Mail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>e-Mail</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rssources.qrc">
    <normaloff>:/images/logo_erp.png</normaloff>:/images/logo_erp.png</iconset>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>381</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="labelSujet">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Sujet:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="Valsujet">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Séparez les adresses par un &lt;span style=&quot; font-weight:600;&quot;&gt;;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="ButtonEnvoie">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>430</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Envoyer</string>
   </property>
  </widget>
  <widget class="QLabel" name="labeldest">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>73</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Destinataire:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMsg">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Message:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboMail">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>94</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>@gmail.com</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>@yahoo.com</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>@yahoo.fr</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>@esprit.tn</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>@outlook.com</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="ButtonAnnuler">
   <property name="geometry">
    <rect>
     <x>294</x>
     <y>430</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Valdest">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>201</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="ValMsg">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>381</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonAttach">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Attachements</string>
   </property>
   <property name="icon">
    <iconset resource="rssources.qrc">
     <normaloff>:/images/attachment.png</normaloff>:/images/attachment.png</iconset>
   </property>
  </widget>
  <widget class="QLineEdit" name="Valpwd">
   <property name="geometry">
    <rect>
     <x>262</x>
     <y>50</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_pwd">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Mot de passe:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ValExp">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>211</width>
     <height>24</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelexp">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Expéditeur:</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listattach">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>361</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="rssources.qrc"/>
 </resources>
 <connections/>
</ui>
